### ISSUE TITLE
Dark background - see nothing

### ISSUE THREAD

**Issue by eyemiru** [Gnurou/tagainijisho#113]
Hey guys, a friend of mine recommended this program, so I wanted to test it.

Sadly, due the cause of my dark background I can't see anything.
I wish backgrounds (the type field as well like in JGlossator) and font colours could be customizable ... that would be really great.

Thanks

Pic of program: http://puu.sh/a8WVP/a4132f5494.png
http://puu.sh/a8XYP/e55c5f77cc.png

**Comment by igyftos** [comment]
The `detailed_default.css` works for me properly as well, however it does not seem to work from `~/.config/tagaini.net/detailed_default.css`, but rather from `/usr/share/tagainijisho/detailed_default.css` for some reason.

:+1:

**Comment by Gnurou** [comment]
Ok, so I had to switch to a dark theme myself to seriously address this issue. >_<
I have pushed a few commits that fix this in the qt5 branch. They make everything look good with both light and dark color schemes and will be included in the next release. Was I less neglectful with my use of colors, this problem would not have happened at all - apologies for that and for taking 2 years to fix such a trivial issue.
